defaults.json

```
@@ -5,5 +5,5 @@
 		"default": "node",
 		"node": "https://nodejs.org/dist/"
 	},
-	"bootstrap": "node/16.18.1"
+	"bootstrap": "node/16.19.1"
```

As of 2023-07-07, there is a newer version of Nodejs available in 16.x.x branch. So, we can update to it.
```suggestion
	"bootstrap": "node/16.20.1"
```

https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V16.md#16.20.1